word_counter.rb

```diff
@@ -0,0 +1,11 @@
+
+def word_count(input_file)
+
+	text = IO.read(input_file).gsub(/\W/, ' ').split(' ')
```

Comment:
If you are using sublime there is another line you have to add beyond indent size:
"translate_tabs_to_spaces": true

If you are using another text editor there is probably a similar setting somewhere.